mantidproject/mantid
# Fix and re-enable Jupyter Notebook shortcut on Windows

**Note: This has been based on a branch forms a part of ~~#33889~~**

**Description of work.**

The NSIS script for the new package based on conda-build (#33742) had disabled the notebook shortcut as it was not working.

This fixes the command and also modernizes it to use jupyter and not IPython. This removes a deprecation notice.

This also fixes the broken Windows icons for desktop and start menu shortcuts.

**To test:**

See #33742 and build a standalone package having checked out this branch. Note that it does not matter if the conda channel is set to `nightly`

Fixes #33812
Fixes #33869 

*This does not require release notes* because **there should not be a change from a users perspective**



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
